src/__test__/redux/actions/samples/updateSample.test.js

```diff
@@ -33,6 +39,16 @@ describe('updateSample action', () => {
     },
   };
 
+  beforeEach(() => {
+    jest.clearAllMocks();
+
+    saveSamples.mockImplementation(() => async () => { });
+
+    enableFetchMocks();
+    fetchMock.resetMocks();
+    fetchMock.doMock();
```

Comment:
are these all required? just mentioning because I've seen some tests where many of these are just copy pasted but aren't needed. What does `doMock` do by the way?

Reply:
in this case yes, we are mocking the fetch for api v2 so we can say that the request succeeeded